tests/test-addons.py

```diff
@@ -47,6 +48,12 @@ def test_basic(self):
         print("Enabling DNS")
         microk8s_enable("dns")
         wait_for_pod_state("", "kube-system", "running", label="k8s-app=kube-dns")
+        print("Enabling Ambassador")
+        microk8s_enable("ambassador")
```

Comment:
ambassador is amd64 only? If so, please move its tests to its own unit test and mark it so it does not run in non-amd64 architectures.